book/01-exploring-the-data.md

```diff
@@ -34,11 +34,8 @@ def metadata_from_url(url):
             f.write(data.content)
             f.flush()
             fmt_metadata = pd.read_csv(f.name, sep='\t').set_index('SampleID')
-            # drop #q2:types from the metadata
-            fmt_metadata = fmt_metadata.drop(['#q2:types'])
             result = qiime2.Metadata(fmt_metadata)
-
-        return result
+            return result
```

Comment:
Is there a reason we decided to do this by hand instead of using `init_metadata_from_url`?

Reply:
I don't see how this could cause the column type issue but who knows